lib/solidus_dev_support/templates/extension/bin/sandbox.tt

```diff
@@ -50,7 +50,7 @@ fi
 cd ./sandbox
 cat <<RUBY >> Gemfile
 gem 'solidus', github: 'solidusio/solidus', branch: '$BRANCH'
-gem 'solidus_auth_devise', '>= 2.1.0'
+<%= "gem 'solidus_auth_devise', '>= 2.1.0'" unless file_name == 'solidus_auth_devise' %>
```

Comment:
What about removing this line and switching `--with-authentication` to `true` in the solidus installer?

https://github.com/solidusio/solidus_dev_support/blob/e5fec9c039607ab9e97c3f821cc49f9bbec08796/lib/solidus_dev_support/templates/extension/bin/sandbox.tt#L74

Reply:
Good call, I'll try that

Reply:
@waiting-for-dev done, an explanation on why we didn't do it before is in the last commit, but basically it boils down to 3017ed64.